CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
   `generate_for` argument to include files with mock generating annotations.
 * Require Dart >= 2.17.0.
 * Require analyzer 5.2.0.
+* Add new `thenReturnInOrder` method to mock multiple calls to a single method in order.
```

Comment:
Ouch, that's a wrong place, we are not adding this new method to the already released version :) Please move up to the -wip section.

Reply:
Could you also please do me a favor and squash it all in one commit with the informative commit message (like the one you used for this PR).